Podcast/MiniPlayerView.swift

```diff
@@ -52,15 +52,16 @@ class MiniPlayerView: UIView {
         miniPlayerSlider = UISlider(frame: .zero)
         miniPlayerSlider.minimumTrackTintColor = .sea
         miniPlayerSlider.maximumTrackTintColor = .silver
-        miniPlayerSlider.thumbTintColor = .clear
+        miniPlayerSlider.isUserInteractionEnabled = false
+        miniPlayerSlider.setThumbImage(UIImage(), for: .normal)
         addSubview(miniPlayerSlider)
         miniPlayerSlider.snp.makeConstraints { make in
             make.top.equalToSuperview()
             make.width.equalToSuperview()
             make.height.equalTo(miniPlayerSliderHeight)
         }
         
-        arrowButton = UIButton(frame: .zero)
+        arrowButton = UtilityButton(frame: .zero)
         arrowButton.setBackgroundImage(#imageLiteral(resourceName: "backArrowDown"), for: .normal)
```

Comment:
can you message Eileen for a darker backArrowDown and backArrowUp icon and replace these?
Also message her for the transparent play and pause button backgrounds and include that in this PR!

Reply:
eileen's having trouble exporting the play/pause with transparent buttons for some reason so i'm gonna leave them in for this pr